test-app/src/main/java/net/rubygrapefruit/platform/test/Main.java

```diff
@@ -388,6 +394,74 @@ private static void ls(String path) {
         }
     }
 
+    private static void longPaths() {
```

Comment:
I think this would be better as a unit test, for example in `FilesTest` or perhaps a new `WindowsFilesTest` class. This way we can check that it still works every time we run the tests (eg when releasing).

Reply:
Done. The FilesTest file already had test for this (they were simply skipped on Windows), so I just had to remove those checks and verify the test pass (they do).

I also removed the "longPaths" option from the test app since it is not needed.